packages/flutter/lib/src/cupertino/route.dart

```diff
@@ -1035,6 +1038,7 @@ Future<T> showCupertinoModalPopup<T>({
   ImageFilter filter,
   bool useRootNavigator = true,
   bool semanticsDismissible,
+  RouteSettings settings,
```

Comment:
To keep the naming consistent with the other `showFoo` methods, can we also call this here `routeSettings`?